docs/index.adoc

```diff
@@ -432,15 +430,13 @@ Therefore, this constellation is not applicable to this layered standard and the
 
 Limitations for binary FMUs:
 
-* Normally only variables in the memory segments owned by the main FMU binary `{.dll, .so}` are accessible by the XCP service, i.e. variables defined in source code modules and in statically linked libraries.
+* Normally only variables in the memory segments owned by the main FMU binary `{.dll, .so}` are accessible by the XCP service, i.e., variables defined in source code modules and in statically linked libraries.
 Accessing variables in the memory of indirectly loaded dynamic libraries requires a special implementation which is out-of-scope for this layered standard.
 
-Limitations for variant <<External XCP Service,External XCP Service>>:

```

Comment:
We had in mind that there could be a future version of this layered standard that defines a way for the FMU to announce memory segments to the importer. But I'm also fine with removing this limitation.

Reply:
This is NOT the job of this standard. Memory segments are defined inside the a2l file.

Reply:
Yes, you are right. This was more related to source code FMUs, where the memory layout is not known yet. The FMU could announce which memory segments are available and the importer could enhance the A2L file with the MOD_PAR info.
So, the limitation would apply to source code FMUs with external XCP service. But we can also remove it completely.